books/free-programming-books-subjects.md

```diff
@@ -18,6 +18,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Containers](#containers)
 * [Database](#database)
 * [Datamining](#datamining)
+* [DevOps](#devops)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.